bin/helpers

```diff
@@ -128,6 +128,12 @@ install_lxd() (
     # on first invocation
     mkdir -p ~/snap/lxd/common/config/
     touch ~/snap/lxd/common/config/config.yml
+
+    if [ -n "${LXD_SIDELOAD_PATH:-}" ]; then
+        systemctl stop snap.lxd.daemon
+        cp "${LXD_SIDELOAD_PATH}" /var/snap/lxd/common/lxd.debug
+        systemctl start snap.lxd.daemon
+    fi
```

Comment:
I'd propose doing this but before the `waitready` earlier in that function. In the microcloud tests, we do it slightly differently so maybe you'd like to reuse it, https://github.com/canonical/microcloud/blob/main/test/includes/microcloud.sh#L136-L142

Reply:
you can also just do cp, followed by reload rather than stop and start